plugin.rb

```diff
@@ -186,6 +186,21 @@ def self.allowed_group_ids
     include_has_chat_enabled? && object.user_option.chat_sound
   end
 
+  add_to_serializer(:current_user, :chat_drafts) do
+    Draft
+      .select(:draft_key, :data)
```

Comment:
`pluck` would be slightly faster here. Since this query happens on the current_user serializer we should consider it.